README.md

```diff
@@ -1,32 +1,28 @@
 # System **MyCanteen**
 
-## Krótki opis projektu
-
-Stworzone oprogramowanie ma być odpowiedzialne za zarządzanie ruchem w stołówkach, ich opiniowaniem, usprawnianiem pracy pracowników w stołówkach oraz ułatwieniem klientom stołówek pozyskiwania zarówno informacji dotyczących aktualnego stanu stołówek, jak i zasięgania opinii innych użytkowników dotyczących wszelakich potraw serwowanych w stołówkach.
-
-## Funkcjonalności stworzonego oprogramowania
-
-Stworzony system ma w przyjazny dla użytkowników sposób umożliwiać im:
-
-1. Śledzenie aktualnego stanu stołówek:
-    * zbieranie informacji na temat aktualnego zatłoczenia w pomieszczeniach stołówki z wykorzystaniem kamer internetowych
-    * zbieranie informacji na temat aktulanego menu w stołówkach od użytkowników (rozpoznawanie tekstu ze zdjęć menu) jak i możliwość dodawania aktualnego menu przez osoby prowadzące lokale
-    * zbieranie informacji na temat aktualnych długości kolejek w lokalach, średniego czasu oczekiwania na posiłek
-2. Zbieranie danych od użytkowników aplikacji:
-    * wystawianie opinii na temat posiłków serwowanych w stołówkach każdego dnia
+System MyCanteen jest przeznaczony dla klientów oraz właścicieli stołówek. Klienci przed wyjściem na obiad będą mogli sprawdzić aktualne obłożenie stołówki, oferowane menu, a także ocenić serwowane potrawy. Właściciel stołówki może przeglądać zebrane informacje, zapoznać się z historycznymi trendami odwiedzin, a także przeglądać informacje zwrotne.
+
+## Zakres funkcjonalny
+
+1. Śledzenie liczby użytkowników:
+    * zbieranie informacji o aktualnym zatłoczeniu stołówki
+    * zbieranie informacji o aktualnych długościach kolejek i średnim czasie oczekiwania na posiłek
+    * podgląd informacji o obłożeniu na żywo
+    * przegląd historii i trendów dla właścicieli
+        * powiadomienia o spodziewanych skokach liczby użytkowników
+        * szacowanie ilości zamówionych dań 
+2. Obsługa menu:
+    * dodawanie menu przez użytkowników i właścicieli stołówek
+    * podgląd oferty dla użytkowników
+3. Zbieranie informacji zwrotnej od użytkowników:
+    * wystawianie opinii na temat serwowanych posiłków
     * prowadzenie statystyk ogólnego zadowolenia klientów z lokali
```

Comment:
@avan1235 Nie rozumiem jak to by miało działać. Po prostu możliwość napisania opinii?

Reply:
Tak, opeiniowanie, gwiazdkowanie, udostępnianie, itd. Po prostu nwm czemu ten dokument ma zajmować tak dużo miejsca jak to możan tak wypunktować więc starałem się troszkę dokładniej coś pisać